limiter_test.go

```diff
@@ -54,32 +55,50 @@ func TestLimit(t *testing.T) {
 func TestWithIncrement(t *testing.T) {
 	type test struct {
 		name          string
+		increment     int
 		requestsLimit int
-		windowLength  time.Duration
 		respCodes     []int
 	}
 	tests := []test{
 		{
-			name:          "no-block",
+			name:          "no limit",
+			increment:     0,
 			requestsLimit: 3,
-			windowLength:  4 * time.Second,
-			respCodes:     []int{200, 200, 429},
```

Comment:
This test has changed:

with `increment = 2` and `limit = 3`,

the httprate middleware used to allow two requests in row

but with this change, it only allows one request